core/src/com/group66/game/cannon/Ball.java

```diff
@@ -384,4 +384,46 @@ public void draw(SpriteBatch batch, float delta) {
         batch.draw(tr, hitbox.x - this.radius, hitbox.y - this.radius,
                 this.radius * 2, this.radius * 2);
     }
+    
```

Comment:
As we stated in issue #53 that we need to separate `update` and `draw` this is OK for now, in the future we should overload en remove duped code